shared/src/packages/orders/libs/types/order-entity.type.ts

```diff
@@ -9,10 +13,15 @@ type OrderEntity = {
   endPoint: string;
   status: OrderStatusValues;
   userId: number | null;
-  businessId: number;
-  driverId: number;
+  businessId: number | null;
   customerName: string | null;
   customerPhone: string | null;
+  shift: { id: number };
```

Comment:
why it's object? 
either shiftId: number or shift: ShiftEntity

Reply:
Because later I am pitting into this object driver and truck for response. But ok, I'll change it.